libs/components/src/layout/layout.stories.ts

```diff
@@ -0,0 +1,67 @@
+import { RouterTestingModule } from "@angular/router/testing";
+import { Meta, StoryObj, componentWrapperDecorator, moduleMetadata } from "@storybook/angular";
+
+import { I18nService } from "@bitwarden/common/platform/abstractions/i18n.service";
+
+import { CalloutModule } from "../callout";
+import { NavigationModule } from "../navigation";
+import { I18nMockService } from "../utils/i18n-mock.service";
+
+import { LayoutComponent } from "./layout.component";
+
+export default {
+  title: "Component Library/Layout",
+  component: LayoutComponent,
+  decorators: [
+    componentWrapperDecorator(
+      /**
+       * Applying a CSS transform makes a `position: fixed` element act like it is `position: relative`
+       * https://github.com/storybookjs/storybook/issues/8011#issue-490251969
+       */
+      (story) =>
+        `<div class="tw-scale-100 tw-border-solid tw-border-2 tw-border-[red]">${story}</div>`
+    ),
+    moduleMetadata({
+      imports: [NavigationModule, RouterTestingModule, CalloutModule],
+      providers: [
+        {
+          provide: I18nService,
+          useFactory: () => {
+            return new I18nMockService({});
+          },
+        },
+      ],
+    }),
+  ],
+} as Meta;
+
+type Story = StoryObj<LayoutComponent>;
+
+export const Empty: Story = {
+  render: (args) => ({
+    props: args,
+    template: `
+        <bit-layout></bit-layout>
+    `,
+  }),
+};
+
+export const WithContent: Story = {
+  render: (args) => ({
+    props: args,
+    template: /*html*/ `
```

Comment:
This needs to be uppercase to work.

```suggestion
    template: /* HTML */ `
```

Reply:
@Hinton Are you using a specific vscode extension? I am using [Inline HTML](https://marketplace.visualstudio.com/items?itemName=pushqrdx.inline-html), which expects a lowercase comment. 

Perhaps we could add this (or a different extension) as a recommended workspace extension?

Reply:
Prettier handles it if it's uppercase and has spaces. https://github.com/prettier/prettier/blob/feb6b519ee0e23eeb272227825ad84ecf1af05ab/src/language-js/embed.js#L289. An alternative is to define a html template literal i.e.
```ts
html`
```